stratum/lib/p4runtime/BUILD

```diff
@@ -1,5 +1,7 @@
+# stratum/lib/p4runtime/BUILD
```

Comment:
Is this a requirement of the build system to have first line (insead of the usual copyright statements?)

Reply:
I frequently add identifying comments to the top of CMakeLists.txt and BUILD files. It helps me keep track of which of the identically-named files I'm working on.